### PR TITLE
Add new website initial work

### PR DESCRIPTION
**Pull request by leandrolanzieri**
## Contribution description
As discussed during last Maintainers weekly meeting, this PR is adding the initial work towards the new design for riot-os.org. You can see the current version in http://demo.alamos.cc/.

With the current Github workflow file the actions will fail, because the needed secrets for deployment and API usage are not there, but I think we should take care of cleaning that up in a different PR if possible.

### CODE REVIEW

File: _data/riot_drivers.json [hunk 1]
```diff
@@ -0,0 +1 @@
...QMC5883L 3-Axis Digital Magnetic Sensor"}, {"file": "group__auto__init__saul", "name": "SAUL Auto Initialisation"}, {"file": "group__drivers__saul__init__devs", "name": "SAUL Device Initialisation"}, {"file": "group__drivers__sht3x", "name": "SHT3x Humidity and Temperature Sensor Series"}, {"file": "group__drivers__shtc1", "name": "SHTC1 Temperature and humidity sensor"}, {"file": "group__drivers__sps30", "name": "SPS30 Particulate Matter Sensor"}, {"file": "group__drivers__hts221", "name": "ST HTS221 digital Humidity Sensor"}, {"file": "group__drivers__si1133", "name": "Si1133 UV Index/Ambient Light Sensor with I2C"}, {"file": "group__drivers__si114x", "name": "Si1145/6/7 UV/Ambient light/Proximity sensors"}, {"file": "group__drivers__si70xx", "name": "Si7006/13/20/21 temperature and humidity sensors"}, {"file": "group__drivers__tcs37727", "name": "TCS37727 RGB Light Sensor"}, {"file": "group__drivers__tmp00x", "name": "TMP006/TMP007 Infrared Thermopile Sensor"}, {"file": "group__drivers__tsl2561", "name": "TSL2561 illuminance sensor"}, {"file": "group__drivers__veml6070", "name": "VEML6070 UV sensor"}, {"file": "group__drivers__ph__oem", "name": "pH OEM sensor device driver"}]}]}
```
Comment (miri64) [inline _data/riot_drivers.json:1]: Why are cpus and boards a `yaml` file and `drivers` are JSON?

File: _data/riot_boards.yml [hunk 2]
```diff
@@ -0,0 +1,416 @@
+- name: Arduino Duemilanove
```
Comment (miri64) [inline _data/riot_boards.yml:1]: This file is generated during deployment as far as I can tell, so should it really be added to the source?

File: _data/riot_cpus.yml [hunk 3]
```diff
@@ -0,0 +1,58 @@
+-  name: Atmel ATmega1281
```
Comment (miri64) [inline _data/riot_cpus.yml:1]: Same for this file.

File: _data/contributors.json [hunk 4]
```diff
@@ -0,0 +1,2102 @@
+[
```
Comment (miri64) [inline _data/contributors.json:1]: This one to.

**Review by miri64 — CHANGES_REQUESTED** [review]
Mh okay this got merged faster than I could make my comments... Still, here is as far as I could get...

**Comment by aabadie** [comment]
> this got merged faster than I could make my comments...

Too bad, especially the not so clean git history. I guess that it's ok for an initial work. An you can still PR your change requests :)

**Comment by miri64** [comment]
Yepp, but I think I wait until things settle, this time :-)

**Pull request merged**